sdks/python/apache_beam/runners/interactive/options/interactive_options.py

```diff
@@ -24,6 +24,8 @@
 
 from __future__ import absolute_import
 
+from dateutil import tz
```

Comment:
Are we using timezone from pytz or tz from dateutil?

Reply:
The user can use both. Here internally we use dateutil.tz get the local timezone info.
Externally, the user can use `pytz.timezone` or `dateutil.tz.gettz` because the `to_tz` just needs to be a subclass of datetime.tzinfo.

Added the comments in the exposed API.